common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -381,7 +381,8 @@ def try_load_course(self, course_dir, course_ids=None, target_course_id=None):
             raise exc
         finally:
             if course_descriptor is None:
-                pass
+                if str(target_course_id) == "course-v1:ArbiX+CS101+2014_T3":
+                    logging.info(f"Investigation Log: {target_course_id} : Course Descriptor is None")
```

Comment:
this shouldn't be part of the `if course_descriptor is None:` block. the logs should be immediately after the `finally:` block. 

`logging.info(f"Investigation Log: {target_course_id} : {course_descriptor} ")` something like this.